crates/sui-framework/packages/sui-framework/sources/url.move

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[deprecated(note = b"`sui::url` module is deprecated, use `std::string` instead")]
```

Comment:
Have we tried just deprecating the type instead?

Reply:
Yup. Just updated the PR.